itou/utils/templatetags/matomo_tags.py

```diff
@@ -0,0 +1,13 @@
+from django import template
+
+
+register = template.Library()
+
+
+@register.simple_tag
+def matomo_attributes(category, action, option):
```

Comment:
```suggestion
def matomo_event(category, action, option):
```
Plus précis et ne ferme pas l'utilisation d'autre tag de ce type à l'avenir.